nengo_loihi/hardware/nxsdk_objects.py

```diff
@@ -294,9 +294,12 @@ def set_axons(self, board, n2board, spike_input):
                     taxon_id = int(tsyn_ids[taxon_idx])
                     self.axon_map.setdefault(input_idx, []).append(
                         self.LoihiAxon(
-                            axon_type=axon_type, chip_id=tchip.id,
-                            core_id=tcore.id, axon_id=taxon_id,
-                            atom=spike.atom))
+                            axon_type=axon_type,
+                            chip_id=tchip.id,
+                            core_id=tcore.id,
+                            axon_id=taxon_id,
+                            atom=spike.atom
```

Comment:
If we're doing things this way, can we put a comma on the last line? Just one less change if we add more arguments later.

Reply:
Yes definitely, will add :+1: